examples/global-boundaries.ipynb

```diff
@@ -0,0 +1,648 @@
+    "import geopandas as gpd\n",
+    "import pyogrio\n",
+    "from tqdm import tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f48c544-b999-455e-9ed7-3dc00237e166",
+   "metadata": {},
+   "source": [
+    "- The URL for a large data file of CGAZ at ADM1 Districts level."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "id": "5c128b65-5346-4cc4-a0bb-ae821be3bac1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# URL of the GeoJSON file\n",
+    "url = \"https://github.com/wmgeolab/geoBoundaries/raw/main/releaseData/CGAZ/geoBoundariesCGAZ_ADM1.geojson\"\n",
+    "filepath = \"geoBoundariesCGAZ_ADM1.geojson\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "46cef009-d18d-46d4-bd3a-4295ac4cfdc8",
+   "metadata": {},
+   "source": [
+    "We'll go ahead and fetch the geojson data in that format"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "id": "510879f3-b6b3-475e-9e59-7eef526dbf25",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 361M/361M [05:33<00:00, 1.08MB/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Make a GET request to download the file\n",
+    "# Streaming, so we can iterate over the response and create a progress bar for this large file\n",
+    "response = requests.get(url, stream=True)\n",
+    "\n",
+    "# Sizes in bytes.\n",
+    "total_size = int(response.headers.get(\"content-length\", 0))\n",
+    "block_size = 1024\n",
+    "\n",
+    "with tqdm(total=total_size, unit=\"B\", unit_scale=True) as progress_bar:\n",
+    "    with open(filepath, \"wb\") as file:\n",
+    "        for data in response.iter_content(block_size):\n",
+    "            progress_bar.update(len(data))\n",
+    "            file.write(data)\n",
+    "\n",
+    "if total_size != 0 and progress_bar.n != total_size:\n",
+    "    raise RuntimeError(\"Could not download file\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e42ea3dd-211e-4e5d-8ef5-5b8765c19b30",
+   "metadata": {},
+   "source": [
+    "## Visualize North, Central, and South America\n",
+    "In this demonstration, we're going to use a large bounding box around North, Central, and South America for our visualization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "id": "41b26edb-8375-4118-a850-cf7fd2f002ec",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Use pyogrio engine for an optimization of the time it takes to read this file\n",
+    "# We will also enable use_arrow, for speed\n",
+    "\n",
+    "# this bounding box will visualize the Western Hemisphere - North, Central, and South America.\n",
+    "nh_bbox = (-175.910339,-59.265881,-27.199402,74.449358)\n",
```

Comment:
I'm curious how many coordinates are in the dataset if we remove this bounding box. Is it too much data?

Reply:
It looks like there are ~3224 rows. But some of those rows contain multipolygons, so more than one boundary in one. Others are single polygons with one polygon per admin boundary.

Reply:
Yea actually upon running this a second time there's not a significant difference in the time it takes to load the data with vs without the northern hemisphere bounding box. So I have removed that.